charts/console/templates/service.yaml

```diff
@@ -30,4 +30,5 @@ spec:
       targetPort: {{ include "console.containerPort" . }}
       protocol: TCP
       name: http
+      nodePort: {{ .Values.service.nodePort }}
```

Comment:
Should we gate these if the service.type is nodeport?

Reply:
Also we have service files depending on type in redpanda, should we follow the same approach here?

Reply:
Sure I will update this PR

Reply:
> Also we have service files depending on type in redpanda, should we follow the same approach here?

We have only NOTES.txt that depend on type of the service. This is console chart, so I'm not sure how Redpanda chart applies here?

Reply:
It doesn't, but in that chart we have two different service files, one per each type, just wondering if you think we need to do this here.

Reply:
> It doesn't, but in that chart we have two different service files, one per each type, just wondering if you think we need to do this here.

Can you send link to Github where the service file is defined?

Reply:
https://github.com/redpanda-data/helm-charts/blob/main/charts/redpanda/templates/service.loadbalancer.yaml and
https://github.com/redpanda-data/helm-charts/blob/main/charts/redpanda/templates/services.nodeport.yaml

Reply:
We agree that changes to console can not be guarded by higher level charts. If console would depend on Redpanda then we could create some helper function.